### PR TITLE
* Fixing an error showing the angular distribution

### PR DESCRIPTION
**Pull request by pconesa**
- Removed all code related with Chimera client connections
  - Created axis, spheres and angular distributions files
  - Updating the compatibility to ChimeraX

### CODE REVIEW

**Comment by pconesa** [comment]
We are currently working on adapting all to ChimeraX.

There are 2 branches one in chimera plugin (that installs chimeraX) and another un pwem for this and this code needs them. I'm changing this to Draft until those branches are merged.

**Comment by azazellochg** [comment]
I see, so old chimera won't be supported anymore?

**Comment by pconesa** [comment]
Initially not...we always can rescue old code to maintain both but we
decided to go for the simplest approach.

**Comment by pconesa** [comment]
Rest the code is ready (scipion-em devel has required changes)

File: relion/viewers/viewer_base.py [hunk 1]
```diff
@@ -587,10 +585,17 @@ def _createAngDistChimera(self, it):
                 self.createAngDistributionSqlite(
                     sqliteFn, nparts,
                     itemDataIterator=self._iterAngles(mdOut))
-
-            return ChimeraClientView(volFn,
-                                     angularDistFile=sqliteFn,
-                                     spheresDistance=radius)
+            if hasattr(self.protocol, 'outputVolumes'):
+                vol = self.protocol.outputVolumes.getFirstItem()
```
Comment (azazellochg) [inline relion/viewers/viewer_base.py:589]: @pconesa is it always selecting origin if the first class even if I'm displaying e.g. class 3?

Comment (pconesa) [reply relion/viewers/viewer_base.py:589]: Yes, we discussed that when making the changes and we thought that origin should be the same for all classes.

**Pull request merged**